### ISSUE TITLE
NGINX 1.13 unknown directive "rtmp" in /etc/nginx/nginx.conf

### ISSUE THREAD

**Issue by DanielnetoDotCom** [arut/nginx-rtmp-module#1116]
Congratulations for your work.
I could make it run on a teste server but on a production server I am getting this error...

![image](https://user-images.githubusercontent.com/18009293/30517815-3abd1654-9b41-11e7-84b3-39f6ff07310e.png)

My conf is like this ...
![image](https://user-images.githubusercontent.com/18009293/30517817-58b2b4c0-9b41-11e7-9bfe-df0ca135094e.png)

Please I really need your help, i already take me days to make it work

**Comment by codegastudio** [comment]
Hi, 
Did you recompile nginx with rtmp module on your production server ?
/etc/nginx path is generally the path when you install nginx with packager system.
Regards

**Comment by DanielnetoDotCom** [comment]
OMG, I just realize I have 2 nginx installed, I have no idea how the remove one in /etc/nginx (could you please help me?)

I have 2 more questions ...

1 - when i use the on_publish it renames my live stream name. on the version I have on the test side keeps the same but on the production side changes it to the on_publish url. there is a way to keep the live stream name, not the url?

2 - another thing... looks like i can stream with no problem, but it is not outputting any stream, like does on test side. I onli get a black screen in any player i have tried to retrieve the stream.

please take a look on my stats... 
![image](https://user-images.githubusercontent.com/18009293/30521173-b2a171e2-9b91-11e7-9b30-e6896d1f5299.png)

**Comment by DanielnetoDotCom** [comment]
Hello
I would like to congratulate you for the fantastic work, I will use this project as an official streaming server.
The project I'm working on is YouPHPTube, a YouTube clone and I'm implementing Live Streaming functionality, as it is on youtube.
If you have the time take a look and tell me what you think.
Website: https://www.youphptube.com/
Demo site: https://demo.youphptube.com/
Git: https://github.com/DanielnetoDotCom/YouPHPTube

I finally managed to solve these two problems ...
1- on_publish renames live stream issue, that's why on_publish does not accept http, and my server was redirecting all http requests to https. I created a virtual host that does not do this redirect and everything worked normally

2- The other problem with the black screen I noticed was a problem in the connection, because at another time I tested the same features and everything worked normally.

Now I just need to resolve how to have only one Ngnix installed

Thanks one more time and I wish hear your opinion about the project
